contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -715,7 +716,20 @@
         },
       },
       role: generateGetterSetter('role_visibility'),
-      language: generateGetterSetter('language'),
+      language: {
+        get() {
+          if (this.multipleSelected) {
+            const languages = this.selectedLanguages;
+            return languages.length > 1 ? LanguagesNames.MUL : languages[0] || null;
+          }
+          return this.getValueFromNodes('language');
+        },
+        set(value) {
+          if (value !== LanguagesNames.MUL) {
```

Comment:
I think you could probably tighten the scope of this check, and broaden the user's ability slightly, by having this ignore it if-and-only-if the existing value was also `LanguagesNames.MUL`. That would additionally handle the use case when the user uploads 2 new resources (no language set), and set them both to `LanguagesNames.MUL`. Lastly, it would also prevent a regression from being able to set `LanguagesNames.MUL` on a single node.